src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -177,23 +185,39 @@ public static void autoCompleterLightTheme(final String textToSelect) {
     public static String getConfirmMessageShowed(final WebElement element) {
 
         WAITER.until(ExpectedConditions.alertIsPresent());
-        System.out.println(element.getText() + " <======");
         return element.getText();
 
     }
 
     /**
-     * @param listOfElements is the WebElements lists.
      * @param element        is the content parameter.
+     * @param listOfElements list of elements.
      * @return WebElement .
      */
-    public static WebElement getWebElementFromMainList(final List<WebElement> listOfElements, final String element) {
+    public static boolean istWebElementPresentOnList(final List<WebElement> listOfElements, final String element) {
+        waitTime(2);
+
         return listOfElements
+                .stream()
+                .anyMatch(elementOnList -> elementOnList.getText().equalsIgnoreCase(element));
+    }
+
+    /**
+     * @param element        is the content parameter.
+     * @param listOfElements list of elements.
+     * @return WebElement .
+     */
+    public static WebElement getWebElementFromAList(final List<WebElement> listOfElements, final String element) {
+        waitTime(2);
+
+        WebElement webElement = listOfElements
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)